app/docker/views/services/create/createservice.html

```diff
@@ -101,6 +101,22 @@
             <!-- !port-mapping-input-list -->
           </div>
           <!-- !port-mapping -->
+          <!-- create-webhook -->
+          <div class="col-sm-12 form-section-title">
+            Webhooks
+          </div>
+          <div class="form-group">
+            <div class="col-sm-12">
+              <label class="control-label text-left">
+                Create update webhook
+                <portainer-tooltip position="top" message="Creates a public webhook that can be used to update the service. It will use the latest version of the service image."></portainer-tooltip>
```

Comment:
This tooltip message, "It will use the latest version of the service image" could be misleading or confusing because (unfortunately and inappropriately) `latest` happens to be a well-known default tag for images. Suppose a stack was configured to use an image `foo/bar:baz` which is then resolved to a specific hashed version in a related service as `foo/bar:baz@sha256:12345...`. Inferred *service image* in this case is content-addressable which will never change. Configured service image (in the stack) perhaps have a different tag that `latest` (in this example it is `baz`) so if we are fetching the latest version of that (which most probably the desired one), we should be more clear about it. If however by latest image we mean the `latest` tag, then it might be undesired.

Reply:
What about: `Creates a public webhook that can be used to update the service. Will automatically re-deploy the service using the most up-to-date version of the image associated to the service.` ?

Reply:
How about something like this? `Create a webhook (or callback URI) to automate the update of this service when the associated image is updated in a repository that supports webhooks. Sending a POST request to this callback URL (without requiring any authentication) will pull the most up-to-date version of the associated image and re-deploy this service.` These phrases can be improved grammatically or to make them more compact though.

Reply:
Also, we might also want to add a link to a more detailed documentation page about webhooks in the tooltip.

Reply:
I think that we will settle for `Create a webhook (or callback URI) to automate the update of this service. Sending a POST request to this callback URL (without requiring any authentication) will pull the most up-to-date version of the associated image and re-deploy this service.`

Tooltips is a bad place for documentation links IMO, they should only be used for a short explanation notice about a specific feature. Our user documentation will cover this feature more deeply.

Reply:
For consistency sake let's use *callback URI* in both the places (and not both *URI* and *URL*).